lua/autorun/server/bounties.lua

```diff
@@ -23,39 +27,57 @@ end
 function AddBounty(ply,target,amount)
 	if !ply:IsPlayer() or !target:IsPlayer() or type(amount) != "number" then return end
 	amount = amount > 0 and amount or 0
-	if ply:PS_HasPoints(amount) then
-		SetPlayerBounty(target,GetPlayerBounty(target) + amount)
+	if BountyLimit[ply:SteamID()] == nil then BountyLimit[ply:SteamID()] = 0 end
+	local total = BountyLimit[ply:SteamID()] + amount
+	if ply:PS_HasPoints(amount) and total <= 10000000 then
+		local add = GetPlayerBounty(target) + amount
+		BountyLimit[ply:SteamID()] = BountyLimit[ply:SteamID()] + amount
+		SetPlayerBounty(target,add)
 		ply:PS_TakePoints(amount)
-		BotSayGlobal("[fbc]"..target:Nick().."'s bounty is now [rainbow]"..GetPlayerBounty(target) + amount.." [fbc]points")
+		BotSayGlobal("[fbc]"..target:Nick().."'s bounty is now [rainbow]"..add.." [fbc]points")
+	elseif ply:PS_HasPoints(amount) and total > 10000000 then
+		ply:ChatPrint("[red]You have reached your limit for today")
 	else
 		ply:ChatPrint("[red]You don't have enough points")
 	end
 end
 
 RegisterChatCommand({'bounty','setbounty'},function(ply,arg)
 	local t = string.Explode(" ",arg)
-	local to,c = PlyCount(t[1])
+	local to,c = PlyCount(string.Implode(table.remove(t)))
```

Comment:
doesnt work. table.remove(t) returns the item which was removed and modifies the table in place. so use whatever table.remove returns as the amount of points.